.github/workflows/python_binding_publish.yml

```diff
@@ -0,0 +1,94 @@
+name: Build python binding wheels
+
+on:
+  push:
+    branches:
+      - dev
+      - master
+  pull_request:
```

Comment:
pull_request was only added for testing and should be removed now?

Reply:
Yes agree. We can remain this only for `push` to speed up the workflow.